### ISSUE TITLE
Up/down arrow keys in song select isn't the same as in stable

### ISSUE THREAD

**Issue by un2wo** [ppy/osu#9817]
In osu!stable, when <kbd>↑</kbd>/<kbd>↓</kbd> is used to select the next/previous song, it doesn't expand or play the associated song. Rather, it only hovers over the little card thing, and the user has to press <kbd>←</kbd>/<kbd>→</kbd> to reveal the included difficulties.

osu!lazer is different, in that <kbd>↑</kbd>/<kbd>↓</kbd> can open up the next set of difficulties in the same way that <kbd>←</kbd>/<kbd>→</kbd> would, but I find the constant loading to be annoying while seeking. So, I'd like for lazer to match stable here.

**Comment by peppy** [comment]
I don't believe this will be changing. Are you able to use pageup/pagedown instead?

**Comment by jalienet** [comment]
PageUp/PageDown is a bit farther away from the arrow keys on my keyboard, so it's a bit inconvenient to use. It's also not as precise.

**Comment by Joehuu** [comment]
Already addressed in the upcoming design.
